sources/us/il/edgar.json

```diff
@@ -6,8 +6,7 @@
             "state": "Illinois"
         },
         "country": "us",
-        "state": "il",
-        "county": "👋TODO👋"
```

Comment:
Thanks @iandees, I need to remember to fill in my own template for new sources! 😂

Reply:
😄 no problem! I was doing that to see if there was some weird encoding problem causing the build failure.